Majasdarbs/04-majasdarbs/bootstrap/04-katrina-tiltina/04-KT-CV-adaptivs-dizains/style.css

```diff
@@ -0,0 +1,438 @@
+    color: white;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start; 
+}
+
+.main-note{
+    margin-top: 10vh;
+    margin-left: 10%;
+    margin-right: 10%;
+    
+}
+
+.main-text{
+    font-size: 3rem;
+}
+
+button{
+    font-family: "Montserrat", sans-serif;
+    font-size: 2rem;
+    color: rgb(60, 22, 66);
+    padding: 9px 50px;
+    background-color: rgba(175, 252, 65);
+    border: none;
+    border-radius: 50px;
+    cursor: pointer;
+    transition: all 0.3s ease 0s;
+    margin-top: 30px; 
+}
+
+button:hover{
+    background-color: rgba(8, 99, 117,0.8);
+    color: rgb(175, 252, 65);
+    
+}
+
+/*second part starts here*/
+.benefits-head{
+    font-family: "Montserrat", sans-serif;
+    padding: 3rem;
+    min-height: 30vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    color:rgb(60, 22, 66);
+}
+
+.benefits-head h3{
+    padding: 1rem;
+    font-size: 2rem;
+    font-family:  sans-serif;
+    font-weight: lighter;
+    
+}
+.benefits-head h2{
+    font-family: "Montserrat", sans-serif;
+    padding: 1rem;
+    font-size: 2.8rem;
+    text-decoration: underline;
+    text-decoration-thickness: 0.2rem;
+}
+
+/*card section start here*/
+
+.cards{
+    width:90%;
+    margin:auto;
+    display: flex;
+    min-height: 70vh;
+    align-items: center;
+    flex-wrap: wrap;
+}
+
+.card{
+    text-align: center;
+    background-color: white;
+    flex: 1 1 25rem;
+    min-height: 40vh;
+    margin: 2rem 5rem;
+    box-shadow: 0px 10px 10px rgba(0,0,0,0.5), 0px 20px 20px rgba(0,0,0,0.1);
+    border-radius: 10px;
+}
+
+.cards img{
+    max-width: 15%;
+    margin: 2rem;
+    padding: 0 0.5rem;
+    
+    
+}
+
+.card h4{
+    padding: 3rem;
+}
+
+.card-icon{
+    background: rgb(8, 99, 117);
+    border-top-left-radius: 20px;
+    border-top-right-radius: 20px;
+}
+
+
+/*article starts here*/
+
+
+.article {
+    padding:2em 10%;
+    margin: auto;
+    display:inline-block;
+    
+    color:rgb(60, 22, 66);
+    font-family: sans-serif;
+  }
+
+.article h3{
+    color: rgb(8, 99, 117);
+}
+
+    .aTitleArea {
+      padding: 2em 0;
+      margin-bottom: 1em;
+      text-align: center;
+      overflow: hidden;
+      position: relative;
+    }
+      .aTitle {
+        color:rgb(60, 22, 66);
+        font-size: 2.8rem;
+        font-family: "Montserrat", sans-serif;
+      }
+  
+        .aTitle span{
+          display: inline-block;
+          font-size: 1em;
+          position: relative;
+  
+        }
+        .aTitle span:after, .aTitle span:before{
+          content:"";
+          position: absolute;
+          border-bottom:3px solid;
+          width:4em;
+          margin:0 1.5em;
+          bottom:.5em;
+        }
+        .aTitle span:before{
+          right:100%;
+        }
+        .aTitle span:after {
+          left:100%;
+        }
+
+  
+    .aContent {
```

Comment:
use more meaningful class names. aContent doesn't say anything, until I check the html content